txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -28,8 +37,23 @@ data ToilVerFailure
                   , tOutputSum :: !Integer}
     | ToilInconsistentTxAux !Text
     | ToilInvalidOutputs !Text  -- [CSL-814] TODO: make it more informative
```

Comment:
@neongreen but forget about it, this part will be changed significantly anyway (outputs verification).